src/applications/personalization/profile/components/personal-health-care-contacts/NonVAPatientMessage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NonVAPatientMessage = ({ testId }) => (
+  <>
+    <p data-testid={testId}>
+      Our records show that you don’t currently receive health care benefits
+      from the VA.
+    </p>
+    <p>
+      If you think this is an error, call us at{' '}
+      <va-telephone contact="8008271000" />
+      <va-telephone contact="711">TTY : 711</va-telephone>. We’re here Monday
```

Comment:
This is rendering incorrectly and not displaying the 'TTY : 711' part, and making it look like a big long link with 711 tacked onto the end. The va-telephone component has a tty prop that can be used to resolve this issue.

I would also request that you update the contact number for the 800 number to reference the constants that are available via the design system component library.

[Here is an example of how to use the tty prop and also where the constants can be found for the VA_BENEFITS number](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/components/direct-deposit/legacy/alerts/EduMigrationDowntimeAlert.jsx#L29)

Reply:
thanks